README.md

```diff
@@ -89,9 +89,15 @@ A list of test run definitions are provided as [`testconfig.yaml`](testconfig.ya
   numPods: 7
   duration: 5
   hostNetwork: false
-  encap: vxlan
-  dataplane: iptables
-  iterations: 3
+  iterations: 1
+  leaveStandingConfig: true
+  Perf:
+    direct: true
+    service: true
+    external: true
+    ControlPort: 32000
+    TestPort: 32221
+    ExternalIPOrFQDN: "a6c519e4e9d5a46d795e0d41bf82e93f-1429478103.us-west-2.elb.amazonaws.com"
```

Comment:
what about this structure?
```suggestion
  Perf:
    direct: true
    service: true
    external:
        ControlPort: 32000
        TestPort: 32221
        IPOrFQDN: "a6c519e4e9d5a46d795e0d41bf82e93f-1429478103.us-west-2.elb.amazonaws.com"
```

Reply:
ControlPort and TestPort are not specific to external tests, so we have to move those back to the Perf structure.

Then the advantage of having an `external` structure is mostly lost.